servant-openapi3.cabal

```diff
@@ -88,10 +88,13 @@ library
                      , insert-ordered-containers >=0.2.1.0  && <0.3
                      , lens                      >=4.17     && <5.4
                      , servant                   >=0.17     && <0.21
+                     , servant-server            >=0.17     && <0.21
+                     , servant-client-core       >=0.17     && <0.21
```

Comment:
wow, can we save this package from depending on `server`?

Reply:
Alas no, we're depending on https://github.com/haskell-servant/servant/blob/e07e92abd62641fc0f199a33e5131de273140cb0/servant-server/src/Servant/Server/Internal/ResponseRender.hs#L68

Reply:
Shame...